exchanges/apis/ftx.py

```diff
@@ -13,8 +16,9 @@
 class FTXApi(BaseExchangeApi):
     api_prefix = "api"
 
-    def __init__(self, passphrase=None, key=None, secret=None):
+    def __init__(self, passphrase=None, subaccount=None, key=None, secret=None):
```

Comment:
Weird to have key and secret come after passphrase and subaccount - inconsistent with other apis.

I'd make it:
key, secret, passphrase, subaccount